bin/node/runtime/src/lib.rs

```diff
@@ -467,17 +474,11 @@ parameter_types! {
 	pub const SlashDeferDuration: pallet_staking::EraIndex = 2;
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
 	pub const MaxNominatorRewardedPerValidator: u32 = 256;
-	pub const ElectionLookahead: BlockNumber = EPOCH_DURATION_IN_BLOCKS / 4;
-	pub const MaxIterations: u32 = 10;
-	// 0.05%. The higher the value, the more strict solution acceptance becomes.
-	pub MinSolutionScoreBump: Perbill = Perbill::from_rational_approximation(5u32, 10_000);
-	pub OffchainSolutionWeightLimit: Weight = RuntimeBlockWeights::get()
-		.get(DispatchClass::Normal)
-		.max_extrinsic.expect("Normal extrinsics have a weight limit configured; qed")
-		.saturating_sub(BlockExecutionWeight::get());
 }
 
 impl pallet_staking::Config for Runtime {
+	const MAX_NOMINATIONS: u32 =
```

Comment:
How will it affect the nominators amount?

Reply:
It doesn't affect nominators amount. This variable set how many nominations of validators can select on nominator.
So now, one nominator can vote for 16 different validators. And far as I understand, this number stayed the same. Can you confirm it?

Reply:
We should test it.

Reply:
Now the amount of nominations is set [here](https://github.com/Cerebellum-Network/pos-network-node/pull/141/files#diff-67684005af418e25ff88c2ae5b520f0c040371f1d817e03a3652e76b9485224aR528)
Previosly it was set [here](https://github.com/Cerebellum-Network/pos-network-node/blob/feature/update-to-polkadot-v0-8-29/frame/staking/src/lib.rs#L380)
The constant value stayed the same(16)